react/jsx-indent-props 2 spaces with expression

- eslint v3.19.0
- eslint-plugin-react 6.10.3

![image](https://cloud.githubusercontent.com/assets/1292082/25927659/041f25d2-362a-11e7-8793-079aac8a0cba.png)

```
  111:15  error  Expected indentation of 12 space characters but found 14  react/jsx-indent-props
  112:15  error  Expected indentation of 12 space characters but found 14  react/jsx-indent-props
```

How to configure in the case?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,10 +7,11 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ### Fixed
 * [`no-unused-prop-types`]: Silence false positive on `never` type in TS ([#2815][] @pcorpet)
-* [`jsx-indent-props`]: Apply indentation when operator is used in front of the upper line ([#2808][] @Moong0122)
+* [`jsx-indent-props`]: Apply indentation when operator is used in front of the upper line ([#2808][], [#2820][] @Moong0122)
 * [Deps] update `jsx-ast-utils` ([#2822][] [jsx-eslint/jsx-ast-utils#102][] @ljharb)
 
 [#2822]: https://github.com/yannickcr/eslint-plugin-react/issues/2822
+[#2820]: https://github.com/yannickcr/eslint-plugin-react/pull/2820
 [#2815]: https://github.com/yannickcr/eslint-plugin-react/pull/2815
 [#2808]: https://github.com/yannickcr/eslint-plugin-react/pull/2808
 [jsx-eslint/jsx-ast-utils#102]: https://github.com/jsx-eslint/jsx-ast-utils/pull/102

--- a/tests/lib/rules/jsx-indent-props.js
+++ b/tests/lib/rules/jsx-indent-props.js
@@ -230,6 +230,28 @@ ruleTester.run('jsx-indent-props', rule, {
     ]
   }, {
     code: [
+      '{this.props.test',
+      '  ? <span',
+      '    className="value"',
+      '    some={{aaa}}',
+      '    />',
+      '  : null}'
+    ].join('\n'),
+    output: [
+      '{this.props.test',
+      '  ? <span',
+      '      className="value"',
+      '      some={{aaa}}',
+      '    />',
+      '  : null}'
+    ].join('\n'),
+    options: [2],
+    errors: [
+      {message: 'Expected indentation of 6 space characters but found 4.'},
+      {message: 'Expected indentation of 6 space characters but found 4.'}
+    ]
+  }, {
+    code: [
       '<App',
       '    foo',
       '/>'

